src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -255,6 +286,43 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
             <SortDropdown dropDownOptions={sortDropDownOptions} />
             <CollectionSearch />
           </Row>
+          <Row paddingTop="12" gap="8" flexWrap="wrap">
+            {markets.map((market) => (
+              <TraitChip
+                key={market}
+                value={marketPlaceItems[market as keyof typeof marketPlaceItems]}
+                onClick={() => {
+                  scrollToTop()
+                  removeMarket(market)
+                }}
+              />
+            ))}
+            {traits.map((trait) => (
+              <TraitChip
+                key={trait.trait_value}
+                value={
+                  trait.trait_type === 'Number of traits'
+                    ? `${trait.trait_value} trait${pluralize(Number(trait.trait_value))}`
+                    : trait.trait_value
```

Comment:
Trait chips are now supposed to have the text include the type of trait for example: `Background: green` currently these chips just have the value, ie `green`